### ISSUE TITLE
Unable to save model when no matched ports found

### ISSUE THREAD

**Issue by TACD** [SpineML/SpineCreator#95]
Sometimes when trying to execute I model I encounter a 'No Matched Ports found' warning - I get the same error when trying to save, and even if I close the model and click 'yes' to the dialog asking me to save changes the model is not saved!

**Comment by ajc158** [comment]
Yeah, sometimes the automatic port matching breaks down - if you reselect the afflicted ports then that is a workaround - a fix is pending
